server/server.go

```diff
@@ -188,6 +192,21 @@ func NewServer(logger hclog.Logger, config *Config) (*Server, error) {
 		m.blockchain.SetConsensus(m.consensus)
 	}
 
+	if m.config.Tracker {
+		//	create tracker
+		if m.tracker, err = tracker.NewEventTracker(
+			logger,
+			nil,
+		); err != nil {
+			return nil, err
+		}
+
+		//	start tracker's listening process
+		if err := m.tracker.Start(); err != nil {
+			return nil, err
+		}
+	}
+
```

Comment:
We don't need initialization of Tracker because server does no longer have it.

Reply:
[Resolved](https://github.com/0xPolygon/polygon-edge/pull/386/commits/585d357ff9ff834ac33cc2fd98a060539046f5dc)